Resources/Prototypes/Entities/Objects/Devices/pda.yml

```diff
@@ -795,6 +845,13 @@
     accentVColor: "#447987"
   - type: Icon
     state: pda-ert
+  - type: CartridgeLoader
+    uiKey: enum.PdaUiKey.Key
+    preinstalled:
+    - CrewManifestCartridge
+    - NotekeeperCartridge
+    - NewsReaderCartridge
+    - WantedListCartridge
```

Comment:
Should this just be using alwayspushinheritance if you need to copy-paste these?